lib/os.js

```diff
@@ -54,3 +54,6 @@ if (binding.isBigEndian)
   exports.endianness = function() { return 'BE'; };
 else
   exports.endianness = function() { return 'LE'; };
+
+exports.sizeof = process.binding('sizeof');
+exports.alignof = process.binding('alignof');
```

Comment:
Ok cool, so long as this doesn't have to go through c++ each time it is accessed.